modules/virt-attaching-vm-to-ovn-secondary-nw-cli.adoc

```diff
@@ -28,20 +28,23 @@ spec:
       domain:
         devices:
           interfaces:
-          - name: secondary <1>
+          - name: secondary # <1>
             bridge: {}
         resources:
           requests:
             memory: 1024Mi
       networks:
-      - name: secondary  <2>
+      - name: secondary  # <2>
         multus:
-          networkName: <nad_name> <3>
+          networkName: <nad_name> # <3>
+      nodeSelector:
+        node-role.kubernetes.io/worker: '' # <4>
 # ...
 ----
 <1> The name of the OVN-Kubernetes secondary interface.
 <2> The name of the network. This must match the value of the `spec.template.spec.domain.devices.interfaces.name` field.
 <3> The name of the `NetworkAttachmentDefinition` object.
+<4> Specifies the nodes on which the VM can be scheduled. The recommended node selector value is `node-role.kubernetes.io/worker: ''`.
```

Comment:
Thank you. 

I'm unsure if we should go one step beyond and explain why it is recommended - to apply to all worker nodes in the cluster.

Maybe that's obvious ...

Reply:
I see value in explaining that, but I think it deserves a better place - like a new section about best practice, somewhere... I would not insist on this change in this (already doc-peer-reviewed PR).

Reply:
Totally fair.